validator/src/main.rs

```diff
@@ -488,11 +488,21 @@ fn main() {
         let (signer_service, signer_addr) = LocalVoteSignerService::new(dynamic_port_range);
         (Some(signer_service), signer_addr)
     };
+
     let init_complete_file = matches.value_of("init_complete_file");
+    let verify_ledger = !matches.is_present("skip_ledger_verify");
```

Comment:
Unrelated tag along?

Reply:
Yeah, just shuffling

Reply:
I'm still getting used to the more relaxed commit policy :slightly_smiling_face: